src/components/WelcomePage/WelcomePage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function WelcomePage() {
+  return (
+    <div>
```

Comment:
App.js already has a enclosing `<div classname="App">...<div>`.
Is this additional div here (and in the other pages really necessary? or would a fragment `<>...</>` be enough

Reply:
Good point! Changed div to fragment here and in other components.